### ISSUE TITLE
Vets-api terminal access for Daniel Lim

### ISSUE THREAD

**Issue by danlim715** [department-of-veterans-affairs/va.gov-team#87255]
### Your Name

Daniel Lim

### Your GitHub handle (terminal access is managed through GitHub)

@danlim715

### Your dsvagovcloud AWS user name (if applicable)

Daniel.Lim

### Team, Role, and Company of the target individual

Pension Benefits Team, Full Stack Engineer, Ad Hoc

### Product Manager (PM) name and email

Anh Platt anh.platt@coforma.io

### Product Owner (PO) name and email

Sanja Bajovic Sanja.Bajovic2@va.gov

### COR Name (leave "None" if you listed an AWS user name above)

Eunice Garcia

### Vendor Onboarding Representative (VOR) Name (leave "None" if you listed an AWS user name above)

Coforma - Jeff Langworthy

### Please confirm if you are listed as a team member in Atlas or are listed in Platform Team Roster

- [X] [Team Member in Atlas](https://www.va.gov/atlas/product_directory/team_members)
- [ ] [Platform Team Roster](https://vfs.atlassian.net/wiki/spaces/AP/pages/1908834623/Platform+Roster)

### Please confirm E-QIP Transmittal/Adjudication. This could be a screenshot of your Transmittal/Adjudication email, a link to another request where you've submitted this proof (i.e. AWS/SOCKS), etc.

- [X] I have attached proof that I have either transmitted my E-QIP survey or previously received a favorable decision (by including in "Additional Notes" section below)

### ArgoCD terminal access requested for the following environments

- [ ] dev
- [ ] staging
- [X] sandbox
- [X] production (if requesting, justify prod access in 'Additional Notes' section)

### Additional Notes

![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/54864006/6a72d37f-dc0b-4808-a1ad-5e1dc449b74f)

I already have access to ArgoCD dev and staging environments, but I need sandbox and production as well, in order to query the production database and retrieve the data necessary to process ITFs.

### Additional resources for Support Engineer

- [ ] For users on a VFS team but not listed as a team member in Atlas, they must start the [Platform orientation process](https://depo-platform-documentation.scrollhelp.site/getting-started/Platform-Orientation.1877344532.html)
- [ ] Instructions for handling requests [here](https://vfs.atlassian.net/wiki/spaces/PTST/pages/2183037084/Backend+Support+Responsibilities#Handle-vets-api-ArgoCD-terminal-access) - Support Only

**Comment by davidpetras** [comment]
@danlim715 it appears you already have access to sandbox as well and are just missing `prod` which requires OCTO approval

**Comment by davidpetras** [comment]
Access to `prod` is pending approval by OCTODE, and we'll update you as 
soon as it's available.

**Comment by JonathanKamensDVA** [comment]
Approved.

**Comment by davidpetras** [comment]
@danlim715  You've been granted access to Production, 
with access to Production limited to 6 months, expiring 01/1/25.

Please set a reminder for yourself to return to this Issue and 
request an extension within 1 week of the expiration date. You'll 
receive 1 reminder with a 72 hour window to respond. Failure to 
respond will result in your access being revoked and will require 
you to fill out a new request.

**Comment by danlim715** [comment]
> [@danlim715](https://github.com/danlim715) Your Production access has expired.
> 
> Would you like to request an extension? You have 72 hours to respond, or your access will be revoked and you will need to fill out a new request.

@davidpetras yes please, I would like to request an extension. Do I need to do the same for accessing the lower environments?

**Comment by davidpetras** [comment]
@LindseySaari @ericboehs can one of you please give us a :thumbsup: to extend their access?

**Comment by davidpetras** [comment]
@danlim715 your Production access has been extended for 6 months,
expiring 07/07/25.

Please set a reminder for yourself to return to this Issue and 
request an extension within 1 week of the expiration date. You'll 
receive 1 reminder with a 48 hour window to respond. Failure to 
respond will result in your access being revoked and will require 
you to fill out a new request.
